shubhali/Online_Shopping_Site/register.html

```diff
@@ -0,0 +1,72 @@
+        <script type="application/x-javascript">
+            addEventListener("load", function() {
+                 setTimeout(hideURLbar, 0);
+                },false
+            );
+            function hideURLbar(){
+                window.scrollTo(0,1); 
+            } 
+        </script>
+    </head>
+    <body>
+        <div class="main-w3layouts wrapper">
+            <h1>Creative SignUp Form</h1>
+            <div class="main-agileinfo">
+                <div class="agileits-top">
+                    <form>
+                        <input class="text" type="text" name="Username" placeholder="Username" required="">
+                        <input class="text email" type="email" name="email" placeholder="Email" required="">
+                        <input class="text" type="password" name="password" placeholder="Password" required="">
+                        <input class="text w3lpass" type="password" name="password" placeholder="Confirm Password" required="">
+                        <div class="wthree-text">
+                            <label class="anim">
+                                <div class="container" style="font-size: 18px;">
+                                    <input type="checkbox"style="width:10px;height=20px;" class="checkbox" required="">
+                                    <button type="button" style="font-size: 18px; color: white;" class="btn" data-toggle="modal" data-target="#myModal">
+                                        I Agree To The Terms & Conditions
+                                    </button>
+                                    <div class="modal" id="myModal">
+                                        <div class="modal-dialog">
+                                            <div class="modal-content" style="color: black;">
+                                                <!-- Modal Header -->
+                                                <div class="modal-header">
+                                                    <h4 class="modal-title">Terms and Conditions</h4>
+                                                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                                                </div>
+                                                <!-- Modal body -->
+                                                <div class="modal-body" style="color: black;">
+                                                    All products are subject to availability at the shop where yoy collect your order.Some shops do not sell all products.
+                                                    Images of product and packaging on the online service are exampleonly and may not be identical to the product or packaging you receive from a shops.
+                                                    Difference may be due to your device's of colorsfactors such as the ingrediants used.the supplier,the region of thr country. and the season of the year.
```

Comment:
thr make it the

ingrediants used.the supplier (please check spelling and after full stop new sentence please make it correct)